src/tool_getparam.c

```diff
@@ -736,7 +736,7 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
         global->tracetype = TRACE_BIN;
         break;
       case 'G': /* --npn */
-        config->nonpn = (!toggle)?TRUE:FALSE;
+        warnf(global, "--npn is no longer supported\n");
```

Comment:
this option is most commonly used as --no-npn, so the user may find it confusing to read --npn in that case
```suggestion
        warnf(global, "%s is no longer supported\n",
              (toggle ? "--npn" : "--no-npn"));
```